integration/simple_app_test.go

```diff
@@ -56,19 +53,26 @@ func testSimpleApp(t *testing.T, context spec.G, it spec.S) {
 			Expect(err).NotTo(HaveOccurred())
 
 			image, _, err = pack.Build.
-				WithBuildpacks(nodeURI, yarnURI).
+				WithBuildpacks(
+					nodeURI,
+					yarnURI,
+					buildpackURI,
+					buildPlanURI,
+				).
 				WithNoPull().
 				Execute(name, source)
 			Expect(err).NotTo(HaveOccurred())
 
-			container, err = docker.Container.Run.Execute(image.ID)
+			container, err = docker.Container.Run.
+				WithCommand("ls -alR /layers/paketo-buildpacks_yarn-install/modules/node_modules").
```

Comment:
Can we avoid hardcoding paketo-buildpacks_yarn-install and instead derive the layer name from the buildpack.toml info?